### ISSUE TITLE
[Bug] Intermittent file metadata error

### ISSUE THREAD

**Issue by CodyCBakerPhD** [NeurodataWithoutBorders/nwb-guide#281]
I've seen this error a few times now; it is easily resolved by navigating to a previous page then again proceeding. Pressing the save button does not fix the issue either.

It is only triggerable by proceeding in a quick, minimal, linear way through a pipeline: choose interface, select source data, pre-add all subject metadata from the session manager, and the only value selected or modified on the file metadata page is the required session start time. Then attempting to go to the next page gives

```javascript
Uncaught (in promise) Error: 1 required inputs are not specified properly.
    at JSONSchemaForm.throw (JSONSchemaForm.js?t=1691680583220:339:15)
    at JSONSchemaForm.validate (JSONSchemaForm.js?t=1691680583220:359:32)
    at async GuidedFooter.onNext [as __onNext] (GuidedMetadata.js?t=1691680582966:24:46)
```

Though strangely I am able to generate previews with the single-file preview button

**Comment by CodyCBakerPhD** [comment]
Both/all

**Comment by CodyCBakerPhD** [comment]
I can make a short video if that would help

**Comment by garrettmflynn** [comment]
Found the issue. This is fixed in #261, but I've brought this particular fix to a new PR now.
